### ISSUE TITLE
Saving/Downloading PDF not giving a pop-up select box.

### ISSUE THREAD

**Issue by CryptoSparky** [brave/brave-browser#14164]
## Description 
  When trying to save/download a PDF the *.tmp file is created in the Downloads folder but no pop-up window is available to confirm location or filename.

## Steps to Reproduce
   1. Have a PDF open or linked in Brave and confirm "Ask where to save each file before downloading" is checked in Settings.
   2. Right Click and "Save As" or select Download from the interface.

## Actual result:
  The PDF is saved as a *.tmp file in the default Windows Downloads folder.

## Expected result:
  A pop-up appears asking where the file should be saved.

## Reproduces how often: 
  Every time.

## Brave version (info found on brave://version)

Brave | 1.20.103 Chromium: 88.0.4324.152 (Official Build) (64-bit)
Revision | 6579930fc53b4dc589c042bec9d0a3778326974d-refs/branch-heads/4324@{#2106}
OS | Windows 10 OS Version 2004 (Build 19041.804)

## Other Additional Information:

- Does the issue resolve itself when disabling Brave Shields? No
- Does the issue resolve itself when disabling Brave Rewards? No
- Is the issue reproducible on the latest version of Chrome? No

**Comment by CryptoSparky** [comment]
Quick update with additional occurrences:

1. Tried downloading a .zip file from https://templates.office.com/en-us/templates-for-Excel and the same issue occurred.
2.  Tried downloading a .zip file from https://app.estimateone.com/ and the same issue occurred.

I tried disabling and re-enabling as below to no effect.
![Brave Download Error](https://user-images.githubusercontent.com/37281962/108276462-978b9200-71c3-11eb-9e0c-448e8bb46f60.PNG)
